### DISCUSSION TITLE
Losing my Badges

### DISCUSSION THREAD

**Discussion by HolisticDev1992** [Schweinepriester/github-profile-achievements#684] [Q&A] [answered]
Will I lose my badges if I change my username?

**Comment by bkpecho** [comment, marked as answer]
No, your badges will stay so long as the email address associated with them remains on your account.

**Comment by itspasindu** [comment]
No, changing your username on GitHub should not cause you to lose your badges. your GitHub profile after changing your username to avoid confusion for others who may be trying to find your profile.
